content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -208,65 +216,73 @@ cd C:\o3de-install
 scripts\o3de.bat register --project-path C:\o3de-projects\MyProject
 ```
 
-#### Option 1: Non-monolithic
+Now, you will choose a build type for your project game release layout.
 
-A pre-built SDK engine supports non-monolithic projects by default. As detailed in [Prerequisites](#prerequisites), you should have already created a Visual Studio project for the engine and registered the engine.
+#### Option 1: Monolithic (Game only)
 
-1. Reconfigure your source engine with the `-D` option, `LY_PROJECTS`, pointing to your project's path.
+The most common workflow is to release your game only, in which case you should choose the **monolithic** build. This will build only your game/server launcher and any necessary dependencies.
+
+1. Use CMake to create a Visual Studio project for an engine that supports monolithic projects. Specify a new CMake build directory (`build\windows_mono`) in your pre-built SDK engine directory, separate from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
 
     ```cmd
     cd C:\o3de
-    cmake -B build/windows -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_VERSION_ENGINE_NAME=o3de-install -DCMAKE_INSTALL_PREFIX=C:\o3de-install -DLY_PROJECTS=C:\o3de-projects\MyProject
+    cmake -B build\windows_mono -S . -DCMAKE_INSTALL_PREFIX=C:\o3de-install -DLY_VERSION_ENGINE_NAME=o3de-install -DLY_MONOLITHIC_GAME=1 -DLY_PROJECTS=C:\o3de-projects\MyProject
     ```
+    - Include `-D` option, `LY_PROJECTS`, to point to your project's path.
 
-    {{< note >}}
-Use `Visual Studio 16` as the generator for Visual Studio 2019, and `Visual Studio 17` for Visual Studio 2022. For a complete list of common generators for each supported platform, refer to [Configuring projects](/docs/user-guide/build/configure-and-build/#configuring-projects).
-    {{< /note >}}
-
-1. In your source engine, use CMake to invoke Visual Studio to append non-monolithic release artifacts to the pre-built SDK layout.
+2. In your source engine, use CMake to invoke Visual Studio to append non-monolithic release artifacts to the pre-built SDK layout.
 
     ```cmd
     cd C:\o3de
-    cmake --build build/windows --target INSTALL --config release
+    cmake --build build\windows_mono --target INSTALL --config release
```

Comment:
The build step can also be slightly simplified by using the CMake build presets feature.
```suggestion
    cmake --build --preset windows-mono-install  --config release
```

The `--config` option still needs to be specified because we don't configure it with the the windows-mono-install preset.
(Side Note: We can always add a "windows-mono-install-release" preset that inherits from the [windows-mono-install](https://github.com/o3de/o3de/blob/development/cmake/Platform/Windows/CMakePresets.json#L172) build preset and the [release](https://github.com/o3de/o3de/blob/development/cmake/Platform/Common/CMakePresets.json#L192-L198) build preset to make this even simpler for users)

Reply:
Good call!